src/elements/content-sidebar/__tests__/DocGenSidebar.test.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { MessageDescriptor } from 'react-intl';
+import { render, screen, waitFor, fireEvent } from '../../../test-utils/testing-library';
+
+import { DocGenSidebarComponent as DocGenSidebar } from '../DocGenSidebar/DocGenSidebar';
+import mockData from '../__mocks__/DocGenSidebar.mock';
+
+const intl = {
+    formatMessage: (message: MessageDescriptor) => message.defaultMessage,
+};
```

Comment:
I think we can minimize the intl code a bit 

in DocGenSidebar.tsx: 
use the `useIntl` hook to get intl
remove the intl prop from the `Props`
remove `injectintl` from the default export


in DocGenSidebar.test.tsx: 
delete the intl mock
import from the default instead

I tried the changes already, should work with the suggested changes